src/assets/wise5/authoringTool/addNode/add-your-own-node/add-your-own-node.component.html

```diff
@@ -39,11 +39,12 @@ <h5 i18n>Select components to add to your new step (optional):</h5>
       </div>
     </div>
     <div fxFlex="100" fxFlex.gt-sm="67">
-      <div fxLayout="row wrap" fxLayoutAlign="start start">
+      <div class="component-types" fxLayout="row wrap" fxLayoutAlign="start start">
         <div class="component-type" *ngFor="let componentType of componentTypes">
-          <button mat-stroked-button (click)="addComponent(componentType)">
-            {{ componentType.name }}
-          </button>
+          <component-selector
+            [componentType]="componentType.type"
+            (componentSelectedEvent)="addComponent($event)"
```

Comment:
We already know the component type at this point, so no need to send $event?
```suggestion
            (componentSelectedEvent)="addComponent(componentType)"
```